### ISSUE TITLE
BJP Manifesto - Content Plan

### ISSUE THREAD

**Issue by kjaggu** [reddit-india/indianelectionmeter.github.io#6]
To begin with, we need to identify what kind of content is going to be available on our website. Who will update the promise development and completion, etc. 

For starters, we can refer to the [BJP manifesto highlights](http://www.bjp.org/images/jpg_2014/manifesto_summary_17.04.2014.pdf) for listing out the various promises.

Some points to consider:
- We need to figure out how are we going to consider whether a promise has been fulfilled or not. 
- Who will keep a watch on the various developments?
- Who will have the rights to modify?
- Some promises are half-fulfilled or the bill is passed in a diluted format. How will we mark them?
- Are we going to allow user commenting on each and every promise?

I am sure there are several other queries which will arise. 

Since the Canadian version is a good reference, let's take a look at their promise:

https://www.trudeaumetre.ca/promise/5201

![image](https://cloud.githubusercontent.com/assets/15857646/11169349/eea43d0a-8bda-11e5-9d6d-cc46ffbb6938.png)

It lists out the promise in one line, retrieved gives the source (from the manifesto), and the update so far with the link to a news article followed by comments. We can easily follow the same format.

**Comment by greyhound291** [comment]
For the sake of convenience, how about we assume that BJP manifesto as the source of promises that they have made. So we list them out, categorically, and that's that.

We can then figure out how we track them:

Some are either yes/no like tax reforms and what not. 

Some are more detailed.. Like arms procurement that have a start/middle-working-phase/end that can be tracked using started/wip/complete

Then there are things like highway projects that can be measured and so they can be maybe put in a percentage complete method.

Anyone else want to discuss data types here, do chime in!

**Comment by kjaggu** [comment]
That will be the broad idea. I will compile a list of the promises from the manifesto. We need to identify the categories and sub-categories for each one of them.

**Comment by kjaggu** [comment]
Moreover, we need to ensure that even for a Yes/No response, there is a relevant source for validation.

**Comment by greyhound291** [comment]
News websites and such should be valid enough.. right?

**Comment by kjaggu** [comment]
Yes. But, there are times when the news are skewed. So, my suggestion is to have 2 or 3 verified sources from 'reputed' outlets.

**Comment by okpatil4u** [comment]
That makes sense. As mentioned in another thread, we will need to archive those links as well.

**Comment by okpatil4u** [comment]
@kjaggu @greyhound291 you can use archive.org to add and archive links. Can you start with this google spreadsheet https://docs.google.com/spreadsheets/d/18Xv94E8kqP6JxeFgfBsQetMaT5wPzY4U8Wqui7J8ua4/edit?usp=sharing
We can decide on other contributing strategies very soon.

**Comment by satyarth** [comment]
Progress update: @davistom made an excel of the full BJP manifesto:
https://docs.google.com/spreadsheets/d/1uJHLsrWgvPG-3ARHQJMoCrxnbACFq6B0Klz8SPHREuk/
